resilience.ipynb

```diff
@@ -0,0 +1,204 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Resilience against hardware failures\n",
+    "\n",
+    "Scenario:  We have a cluster that partially consists of preemptible ressources.  That is, we'll have to deal with workers suddenly being shut down during computation.\n",
+    "\n",
+    "Relevant docs: <http://distributed.dask.org/en/latest/resilience.html#hardware-failures>"
```

Comment:
Maybe we should give some examples : dask-jobqueue cluster with jobqueue system walltime, dask-kubernetes with preemptible resources?

Reply:
Yes, we could mention this here.

Reply:
Done.